x/nftbackedloan/types/loan.go

```diff
@@ -0,0 +1,157 @@
+package types
+
+import (
+	time "time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	types "github.com/cosmos/cosmos-sdk/types"
+)
+
+func MinSettlementAmount(bids []NftBid) types.Coin {
+	if len(bids) == 0 {
+		return types.Coin{}
+	}
+	bidsSortedByPrice := NftBids(bids).SortHigherPrice()
+	minimumSettlementAmount := types.NewCoin(bidsSortedByPrice[0].BidAmount.Denom, sdk.NewInt(0))
```

Comment:
this is bad practice.

- if len is 0, emit panic with out of range exception
- return type of min settlement amount should be sdk.Int rather than creating such vulnerability